protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -988,30 +998,24 @@ func templateToParts(path string, reg *descriptor.Registry, fields []*descriptor
 	var parts []string
 	depth := 0
 	buffer := ""
-	jsonBuffer := ""
```

Comment:
This second buffer was just a complexity gainer, I added a single branch to the `}` case, where it extracts the parameter name and camel-case it, when the camel-casing is enabled.